tools/isobuild/linker.js

```diff
@@ -8,6 +8,7 @@ import LRU from 'lru-cache';
 import {sourceMapLength} from '../utils/utils.js';
 import files from '../fs/files.js';
 import {findAssignedGlobals} from './js-analyze.js';
+import {convert} from '../utils/colon-converter.js';
```

Comment:
Maybe `import { convert as converColons } ...` for clarity? The `convert` name by itself is pretty generic.